src/data/faq.json

```diff
@@ -375,20 +374,20 @@
                         "anchor": "admission_policy_2gplus_howto",
                         "active": false,
                         "textblock": [
-                            "If you have stored a valid vaccination or recovery certificate and a valid test certificate in the app, these will also be shown in a combined display from version 2.16 onwards. You can then use the switch to toggle between the QR code of the '2G certificate' (vaccination or recovery certificate) and that of the test certificate to quickly and easily provide 2G+ proof. Both QR codes must be checked on site using the CovPassCheck app together with presentation of the ID card if 2G+ proof is required. For privacy reasons, when the certificates are checked by the CovPassCheck app, it is not apparent whether the individual has a vaccination or recovery certificate. When scanning the QR codes, the checkers only see whether the certificate is valid or not.",
-                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination or recovery certificate and a negative test?</a></li></ul>"
+                            "If you have stored a valid vaccination certificate or a certificate of recovery and a valid test certificate in the app, these will also be shown in a combined display from version 2.16 onwards. You can then use the switch to toggle between the QR code of the '2G certificate' (vaccination certificate or certificate of recovery) and that of the test certificate to quickly and easily provide 2G+ proof. Both QR codes must be checked on site using the CovPassCheck app together with presentation of the ID card if 2G+ proof is required. For privacy reasons, when the certificates are checked by the CovPassCheck app, it is not apparent whether the individual has a vaccination certificate or a certificate of recovery. When scanning the QR codes, the checkers only see whether the certificate is valid or not.",
+                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination certififcate or a certificate of recovery and a negative test?</a></li></ul>"

```

Comment:
"certififcate" typo

Reply:
Thank you! Fixed in [`63367c7`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/63367c733bb083f892e8cba581a5122b81f4ad8e).